### PR TITLE
Gh 2592 - navigating dao

### PR DESCRIPTION
**Pull request by biafra23**
Handles #2592 

Changes proposed in this pull request:
Implementation | Design
----------------|--------
image | image

- Background removed in most places
- Steps removed

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
on the small phone, the "Navigating dao" has too much top vertical spacing

File: Multisig/UI/ClaimToken/NavigatingDAOViewController.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+class NavigatingDAOViewController: UIViewController {
+
+    @IBOutlet weak var firstParagraph: UILabel!
+    @IBOutlet weak var screenTitle: UILabel!
+    @IBOutlet weak var paragraphTitle: UILabel!
+    @IBOutlet weak var secondParagraph: UILabel!
+    @IBOutlet weak var nextButton: UIButton!
+
+    @IBOutlet weak var item1Label: UILabel!
```
Comment (MouazAlzahabi) [inline Multisig/UI/ClaimToken/NavigatingDAOViewController.swift]: I just find the name itemXLabel a bit strange, any better naming for it ?

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift [hunk 2]
```diff
@@ -146,18 +153,48 @@ class ClaimSafeTokenFlow: UIFlow {
...
-        let vc = factory.tokenDistribution { [unowned self] in
-            showDisclaimer()
+    func showWhatIsSafe() {
+        let vc = factory.chooseWhatIsSafe { [unowned self] in
+            showTokenDistribution()
         }
+        show(vc)
+    }
 
+    func showGuardian() {
```
Comment (DmitryBespalov) [inline Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift]: this is wrong, i.e. this code was deleted before and it should use the "SelectDelegate" flow

**Review by DmitryBespalov — CHANGES_REQUESTED** [review]

File: Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift (hunk 2 above)
Comment (DmitryBespalov) [inline Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift]: seems like a merge conflict / wrong merge: this and the following methods should be just deleted

Comment (biafra23) [reply Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift]: I tried my best. Can you verify it is ok now?

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**